### ISSUE TITLE
Line Start hotkey does not work

### ISSUE THREAD

**Issue by AddisonG** [dbeaver/dbeaver#36222]
### Description

I want to set the Dbeaver hotkeys/shortcuts up on my Mac so that they are similar to windows.

However, it is impossible to make the "home" and "end" keys map to "Line Start" and "Line End".

It simply doesn't work, and Dbeaver always maps those keys to "Text Start" and "Text End", even though I mapped them differently.

Here is a screenshot of my current keys:
![image](https://github.com/user-attachments/assets/4c34d19a-b921-42d3-ad2b-edcd0f6c3dc7)

Interestingly, it works fine if I'm holding shift - so selecting to start/end of line, or selecting to start/end of text work great.

Happy to provide video evidence, if people are unable to reproduce.

### DBeaver Version

Version 24.2.4.202411031831

### Operating System

MacOS Sonoma 14.5 M3 Pro

**Comment by E1izabeth** [comment]
Hi, @AddisonG 
Could you check the following answer https://apple.stackexchange.com/a/16137
Does it help?

**Comment by dnicolaev** [comment]
This is a very old issue. I have been facing it for a couple of years already.
After rebinding in DBeaver - it will work as intended for some time. But after intense editing it will default back to "Text Start" and "Text End" - I didn't find a pattern on when exactly it defaults back.

After restarting DBeaver - it goes back to what I set it to be "Line Start/End" - but will go away again after some time.

The answer referred in https://apple.stackexchange.com/a/16137 - yes I made those settings long ago.

DBeaver Version 24.3.0.202412011650
MacOS Sonoma 14.6.1 (23G93)
